templates/inactive_reviewer_needinfo.txt

```diff
@@ -0,0 +1,19 @@
+The following {{ plural('patch is', revisions, 'patches are') }} waiting for review from an inactive reviewer:
+
+| ID  | Title |  Author | Inactive Reviewer |
+| --- | :---- |  ------ | ----------------- |
+{% for rev in revisions -%}
+{% for reviewer in rev['reviewers'] -%}
+{% if rev['reviewers'][0] == reviewer -%}
+| [D{{ rev['rev_id'] }}](https://phabricator.services.mozilla.com/D{{ rev['rev_id'] }}) |  {{ rev['title'] | e }} | [{{ rev['author']['nick'] }}](https://bugzilla.mozilla.org/user_profile?user_id={{ rev['author']['id'] }})  |
```

Comment:
At least for the "unavailable" case, we should give more information and tell the author when the reviewer will be available again (so they know how long it would take if they decided to wait for the reviewer).

Reply:
Done in f6f5a5943c3586ccd5d52c75848d31e8dd041d10